Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -118,18 +128,24 @@ var DatePickerIOS = React.createClass({
 
   render: function() {
     var props = this.props;
+    let minimumDate = props.minimumDate;
```

Comment:
Ok, what you think is : Remove minimumDate and maximumDate native props from render and assign it in _onChange function that's right ? So, did I also remove date prop from render and only assign int inside _onChange ?